### ISSUE TITLE
Implement search

### ISSUE THREAD

**Issue by ferndot** [codexa/firetext#155]
This will make it easier for a user to find documents.

**Comment by ferndot** [comment]
We can use a "pull to search" element for this: http://usehook.com/

**Comment by ferndot** [comment]
Also, [ListJS](http://www.listjs.com) with Fuzzy Search enabled for the actual searching.  I will take this, and implement it in a branch.

**Comment by HR** [comment]
Pull for a search bar to appear?

**Comment by jancborchardt** [comment]
Yes, pulling down for a search bar to appear is a common pattern. Pulling down even further is for refresh.

**Comment by ferndot** [comment]
Adding this to the list for Filebrowser v2.
